server/src/models/people/populate.js

```diff
@@ -30,17 +30,120 @@ const format_csv_records_then_register_to_headcount_models = (
     );
   });
 };
+export default async function ({ models }) {
+  const { EmployeeAgeGroup } = models;
 
-export default function ({ models }) {
-  const headcount_models_and_records = _.map(
-    headcount_csv_names_by_headcount_model_name,
-    (headcount_csv_name, headcount_model_name) => ({
-      model: models[headcount_model_name],
-      records: get_standard_csv_file_rows(`${headcount_csv_name}.csv`),
+  const employee_age_rows = _.chain(
+    get_standard_csv_file_rows("org_employee_age_group.csv")
+  )
+    .reject(["dept_code", "ZGOC"])
+    .map((row) => ({
+      ...row,
+      2016: +row[2016],
+      2017: +row[2017],
+      2018: +row[2018],
+      2019: +row[2019],
+      2020: +row[2020],
```

Comment:
Lots of repetition of these years. All these would need to be maintained on each data update.

I'd prefer to have the csv itself switch from "wide" (years as column headers) to "long" (column for year dimension, row per year) but that can't change until we're actually ready to drop the old `Table` class way of loading this data in the client.

If you look at the current csv's though, you'll see that there's a common pattern to the columns. All of these datasets (small exception for avg age) could be processed by a utility like
```
const process_population_data = (csv_name) => _.chain(
    get_standard_csv_file_rows(csv_name)
  )
    .reject(["dept_code", "ZGOC"])
    .map(({dept_code, dimension, ...data_columns}) => ({
      dept_code,
      dimension,
      ..._.mapValues(data_columns, _.toNumber)
    }))
    .value();
```

Less repetition ([DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)), and no more hard-coded years (also sort of DRY, but more importantly "[single source of truth](https://en.wikipedia.org/wiki/Single_source_of_truth)" for the years from the csv column headers themselves).